README.md

```diff
@@ -101,3 +102,86 @@ func main() {
 Logging is implemented with the standard Golang [slog package](https://pkg.go.dev/log/slog). The logger
 defines interfaces for [Debugf, Infof, and Errorf](./spectator/logger/logger.go). There are useful messages
 implemented at the Debug level which can help diagnose the metric publishing workflow.
+
+---
+
+## Migrating from 0.2.X to 0.3.X
```

Comment:
I think we are going to go for it and make this the 1.0.X version, because it fulfills the vision we have.

Python did not go full 1.0.X yet, because some legacy API cruft remains. We remove that and make the Meter handling `withId` variations the same as here, and I think we can then release that as a 1.0.X.

Reply:
Sounds good. I'll update the README